tests/components/airvisual/test_config_flow.py

```diff
@@ -140,6 +124,23 @@ async def test_migration(hass):
     }
 
 
+async def test_node_pro_error(hass):
+    """Test that an invalid Node/Pro ID shows an error."""
+    node_pro_conf = {CONF_IP_ADDRESS: "192.168.1.100", CONF_PASSWORD: "my_password"}
```

Comment:
This is being reused between tests so can be added to the top as a const. Something like FIXTURE_CONFIG = {CONF_IP_ADDRESS: "192.168.1.100", CONF_PASSWORD: "12345"}

Reply:
Same comment as above.